tiledb/sm/misc/comparators.h

```diff
@@ -46,16 +46,12 @@ namespace tiledb {
 namespace sm {
 
 /** Wrapper of comparison function for sorting coords on row-major order. */
-template <class T>
 class RowCmp {
  public:
-  /**
-   * Constructor.
-   *
-   * @param dim_num The number of dimensions of the coords.
-   */
-  RowCmp(unsigned dim_num)
-      : dim_num_(dim_num) {
+  /** Constructor. */
+  RowCmp(const Domain* domain)
+      : domain_(domain) {
+    dim_num_ = domain->dim_num();
```

Comment:
just curious: what's the motivation for removing this from the list initialization? its not a big deal for a primitive

Reply:
Just slipped, I will add it to the list initialization.